modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+An alert, `NetObservAgentFlowsDropped`, is triggered when the Network Observability eBPF agent hashmap table is full or when the capacity limiter is triggered. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
+
+[NOTE]
+====
+Increasing the `cacheMaxFlows` might increase the memory usage of the eBPF agent. 
+====
+
+.Procedure
+
+. In the web console, navigate to *Operators* -> *Installed Operators*.
```

Comment:
The ISG says not to use **navigate**, rather use _go to_. Probably not worth holding up the PR, just for future reference.